src/components/WwwFrame/PromotionalBanner/Banners/AppealBanner/AppealBanner15.vue

```diff
@@ -138,18 +119,41 @@ export default {
 		},
 	},
 	created() {
-		// Check to make sure we're on the client to prevent flash of unwanted banner state
-		if (!this.$isServer) {
-			if (store2.session.get('appeal_banner_15_shrunk')) {
-				this.open = false;
-			} else {
-				this.open = true;
-			}
+		if (cookieStore.get('appeal_banner_15_shrunk')) {
+			this.open = false;
+		} else {
+			// open banner
+			this.open = true;
 		}
 	},
+	mounted() {
+		// Get recent funded loans data
+		this.apollo.query({
+			query: recentFundedLoans,
+			variables: {
+				start: '2020-10-01'
+			}
+		}).then(({ data }) => {
+			// Default to 1 to avoid division by 0
+			const fundedGoal = this.appealBannerContent?.dataObject?.kiva15FundedLoansGoal || 1;
+			const numRecentFundedLoans = data?.general?.kivaStats?.numRecentFundedLoans || 0;
+			if (numRecentFundedLoans !== 0) {
+				const vm = this;
+				const percentFull = { val: 0 };
+
+				gsap.to(percentFull, 1, {
+					val: (numRecentFundedLoans / fundedGoal) * 100,
+					roundProps: 'val',
+					onUpdate() {
+						vm.fifteenYearGoalPercent = percentFull.val;
+					}
+				});
+			}
+		});
+	},
 	methods: {
 		shrinkAppeal() {
-			store2.session.set('appeal_banner_15_shrunk', true);
+			cookieStore.set('appeal_banner_15_shrunk', true);
```

Comment:
I would recommend adding a third argument here with `{ path: '/' }` to ensure that only one cookie is created for the whole site instead of a cookie for each page